packages/lab/src/common-hooks/calcPreferredHeight.js

```diff
@@ -0,0 +1,36 @@
+export const calcPreferredHeight = (props = {}) => {
```

Comment:
this will disappear when Tree gets finished. This was originally used by List - that code has now been refactored into useListHeight. I suspect Tree can also be refactored to use useListHeight. -at which point we will probably want to rename it to useComponentHeight or useCollectionHeight or something similar.
I'll add a comment to this effect in calcPreferredHeight